Bottom: ca034ec8341463fdbcc621543baf3e74177e5b18
Top:    c08cde29343f71de280151c97c337795541cad90
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:57:51 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index b2add35..ff94ccf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
